RecoEgamma/EgammaTools/interface/EgammaDNNHelper.h

```diff
@@ -49,8 +49,9 @@ namespace egammaTools {
     // which has access to all the variables.
     std::pair<uint, std::vector<float>> getScaledInputs(const std::map<std::string, float>& variables) const;
 
-    std::vector<std::vector<float>> evaluate(const std::vector<std::map<std::string, float>>& candidates,
-                                             const std::vector<tensorflow::Session*>& sessions) const;
+    std::vector<std::pair<uint, std::vector<float>>> evaluate(
```

Comment:
not for this PR, but I think the same comment applies here as was suggested at the DeepSC PR:
these kind of supernested structures are easy to write down but hard to reason about later. It would be better to define classes that encapsulate the required data.